charts/rancher-webhook/templates/rbac.yaml

```diff
@@ -10,3 +10,16 @@ subjects:
 - kind: ServiceAccount
   name: rancher-webhook
   namespace: {{.Release.Namespace}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: rancher-webhook-sudo
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
```

Comment:
I'm a bit nervous about adding another pre-built-in binding to cluster-admin - it makes this role an escalation path for an attacker. I think there's not much we can do here since we want this have permission on any types that the webhook validates (otherwise users will be stopped by RBAC even if the WH approves), but would you mind confirming that this is the best we can do here?

Reply:
Discussed offline - this is minimal as far as the permissions that we can assign this.